qa/acceptance-tests/src/test/java/io/camunda/it/tenancy/ClusterVariableTenancyIT.java

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.tenancy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.client.api.command.ProblemException;
+import io.camunda.client.api.search.response.ClusterVariable;
+import io.camunda.qa.util.auth.Authenticated;
+import io.camunda.qa.util.auth.TestUser;
+import io.camunda.qa.util.auth.UserDefinition;
+import io.camunda.qa.util.multidb.MultiDbTest;
+import io.camunda.qa.util.multidb.MultiDbTestApplication;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
+
+@MultiDbTest
+@DisabledIfSystemProperty(named = "test.integration.camunda.database.type", matches = "AWS_OS")
+public class ClusterVariableTenancyIT {
+
+  @MultiDbTestApplication
+  static final TestStandaloneBroker BROKER =
+      new TestStandaloneBroker()
+          .withBasicAuth()
+          .withMultiTenancyEnabled()
+          .withAuthenticatedAccess();
+
+  private static final String VALUE_RESULT = "\"%s\"";
+  private static final String ADMIN = "admin";
+  private static final String USER1 = "user1";
+  private static final String USER2 = "user2";
+  private static final String TENANT_A = "tenantA";
+  private static final String TENANT_B = "tenantB";
+
+  private static final String TENANT_A_VAR_NAME = "clusterVarTenantA";
+  private static final String TENANT_A_VAR_VALUE = "tenantAValue";
+  private static final String TENANT_B_VAR_NAME = "clusterVarTenantB";
+  private static final String TENANT_B_VAR_VALUE = "tenantBValue";
+
+  @UserDefinition
+  private static final TestUser ADMIN_USER = new TestUser(ADMIN, "password", List.of());
+
+  @UserDefinition
+  private static final TestUser USER1_USER = new TestUser(USER1, "password", List.of());
+
+  @UserDefinition
+  private static final TestUser USER2_USER = new TestUser(USER2, "password", List.of());
```

Comment:
💭 We could add global cluster variables as well to test that they work in combination with multi-tenancy. USER2 could be able to READ global variables, for example. That way, we ensure such a user can really only see global variables, but not tenant variables by accident.